docs/getting-started.md

```diff
@@ -0,0 +1,629 @@
+            id="content_image"
+            class="card-img-top"
+            style="max-height: 220px; width: auto"
+            src="%image_id%"
+            alt="Card image cap"
+          />
+          <div class="card-body">
+            <p class="card-text text-muted" style="font-weight: bold">
+              %text%
+            </p>
+            <p class="card-text text-muted">
+              %year%
+            </p>
+            <p class="card-text text-muted">
+              "%caption%"
+            </p>
+          </div>
+        </div>
+```
+
+```javascript
+     const render = (data, config) => {
+       const template = document.getElementById('content_template').innerHTML
+       const html = template
+         .replace('%item_id%', 'row-' + data.imdbID)
+         .replace('%image_id%', data.Poster)
+         .replace('%text%', data.Title)
+         .replace('%data%', JSON.stringify(data, null, 2))
+         .replace('%year%', data.Year)
+         .replace('%caption%', (config.caption ? config.caption : ""))
+
+       const element = document.createElement('div')
+       document.getElementById('search_content').appendChild(element)
+       element.outerHTML = html
+
+       if (config && (config.show_poster === false)) {
+         document.getElementById('content_card').removeChild(
+           document.getElementById('content_image')
+         )
+       }
+     }
+
+```
+
+Once again, we can test using the test tool.
+
+
+### 6. Deploy code to a development environment.
+
+To wire these panels up in Composer, you'll need to host them on a public domain. Just zip up a bundle like you usually would and deploy to a test environment.
+
+Once they're uploaded, you can wire up Composer to use the panels in the settings page. See [these instructions](https://redirector.arcpublishing.com/alc/arc-products/ellipsis/user-docs/ellipsis-custom-embed-power-ups-for-rich-third-party-content/#Configuring-a-Custom-Embed-Integration).
+
+Save a few documents with the embed in them, then go look at those documents published on the web.
+
+### 7. Create an appropriate feature component in PageBuilder Fusion.
+
+Wait...there's nothing there? Ah, geeze, we forgot to write a Fusion feature for actually rendering these things!
+
+Fortunately it's not too hard to add. We already have the right content sources configured, after all.
+
+Let's update our movie-detail component (from the Fusion recipes) to utilize our content source and configuration options.
+
+```javascript
+@Consumer
+class MovieDetail extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      movie:  {}
+    }
+    this.fetch = this.fetch.bind(this)
+    this.fetch()
+  }
+
+  fetch() {
+    const { movie } = this.state
+    const { imdbID, caption, show_poster } = this.props
+    this.fetchContent({
+      movie: {
+        source: 'movie-find',
+        query: { "imdbID": imdbID },
+        transform: (data) => {
+          const movieData = data || {}
```

Comment:
Another way to implement this block might be:

```
return Object.assign(
  {},
  data,
  {
    Poster: show_poster ? data.Poster : null,
    caption: caption
  }
)
```
Might not make a difference, but this has the benefit of creating a new object entirely instead of mutating the original.